firstvoices/backend/views/dictionary_views.py

```diff
@@ -52,3 +57,116 @@ def get_queryset(self):
             )
         else:
             return DictionaryEntry.objects.none()
+
+
+@extend_schema_view(
+    list=extend_schema(
+        description="Returns a word of the day for the given site.",
+        responses={
+            200: DictionaryEntryDetailSerializer,
+            403: OpenApiResponse(description="Todo: Not authorized for this Site"),
+            404: OpenApiResponse(description="Todo: Site not found"),
+        },
+    ),
+)
+class WordOfTheDayView(
+    FVPermissionViewSetMixin,
+    SiteContentViewSetMixin,
+    mixins.ListModelMixin,
+    viewsets.GenericViewSet,
+):
+    """
+    Picks and returns word-of-the-day for a given site.
+    """
+
+    http_method_names = ["get"]
+    serializer_class = DictionaryEntryDetailSerializer
```

Comment:
I'm thinking about future enhancements to allow writing words of the day, and I think we should update this to use a serializer that is specific to words of the day. It only needs a date field and the word.